Source/cmake/WebKitCommon.cmake

```diff
@@ -67,6 +67,8 @@ if (NOT HAS_RUN_WEBKIT_COMMON)
         set(COMPILER_IS_CLANG ON)
     endif ()
 
+    option(USE_LIBCXX "Use llvm libc++" OFF)
```

Comment:
Mmm, I kinda suspect that most people who want libc++ will either pass the -stdlib=libc++ flag manually, or will get it by default, but I suppose it's harmless. It's surely not needed for porting to loongarch64, though, so I would split this out into a separate commit.

Reply:
Well actually, I see this is only needed if the platform default stdlib is libstdc++. So that means this option ensures that the compiled WebKit is incompatible with other system libraries. It's probably better not to provide this, right? It's an unusual thing to want to do, and you can always pass the flag manually if you really want it.

Reply:
Ah, I see you split this out into https://github.com/WebKit/WebKit/pull/3904.